README.md

```diff
@@ -291,19 +291,24 @@ meta_gamma(meta, 0:2)
 
 
 ## Genetic diversity
-Multiallelic data can currently only be dealt with by the GitHub version of rdiversity.
 1. Initialise data:
 ```{r}
 library()
```

Comment:
Surely library(rdiversity)?
And comment that pinfsc50 must be installed for this example to work...